qiskit/pulse/instruction_schedule_map.py

```diff
@@ -161,39 +169,89 @@ def get(self,
 
         Returns:
             The Schedule defined for the input.
+
+        Raises:
+            PulseError: When invalid parameters are specified.
         """
         instruction = _get_instruction_string(instruction)
         self.assert_has(instruction, qubits)
-        schedule_generator = self._map[instruction].get(_to_tuple(qubits))
+        schedule_param_tuple = deepcopy(self._map[instruction][_to_tuple(qubits)])
```

Comment:
nitpick: It is a little challenging keeping `schedule.parameters`, `params`, and `schedule_param_tuple.parameters` straight.

```
schedule  # this is clear
schedule.parameters   # this seems clear
params     # input_params? as an alias within the function?
kwparams  # seems clear that its keyword args
schedule_param_tuple.parameters = schedule_arguments  # ?? I'm not sure
```